domain/accounts_conversor.py

```diff
@@ -0,0 +1,6 @@
+from domain.bankaccount import BankAccount
+from domain.ledgeraccount import LedgerAccount
+
+
+def get_stage_account(bank_acc: BankAccount) -> LedgerAccount:
```

Comment:
the conversor is between an AccountId e LedgerId, not between BankAccount e LedgerAccount